pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServerStatusOp.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+    Button,
+    DialogContent,
+    Grid
+} from "@material-ui/core";
+import Dialog from "../../CustomDialog";
+import React, {useState} from "react";
+import moment from "moment/moment";
+import {RebalanceServerSection} from "./RebalanceServer/RebalanceServerSection";
+import CustomCodemirror from "../../CustomCodemirror";
+import './RebalanceServer/RebalanceServerResponses/CustomCodeMirror.css';
+import {RebalanceServerResponseCard} from "./RebalanceServer/RebalanceServerResponses/RebalanceServerResponseCard";
+import CustomizedTables from "../../Table";
+
+export type RebalanceTableSegmentJobs = {
+    [key: string]: {
+        jobId: string,
+        messageCount: number,
+        submissionTimeMs: number,
+        jobType: string,
+        tableName: string,
+        REBALANCE_PROGRESS_STATS: string,
+        REBALANCE_CONTEXT: string;
+    }
+}
+
+type RebalanceServerStatusOpProps = {
+    rebalanceServerStatus: RebalanceTableSegmentJobs;
+    hideModal: () => void;
+};
+
+export const RebalanceServerStatusOp = (
+    { rebalanceServerStatus, hideModal } : RebalanceServerStatusOpProps
+) => {
+    const [jobSelected, setJobSelected] = useState<string | null>(null);
+
+    const BackAction = () => {
+        return (
+            <Button
+                style={{ textTransform: 'none' }}
```

Comment:
buttons are default capitalized across project. either we should change the default behaviour or keep it as is.

Reply:
Kept it as it is!